addons/flannel/0.21.2/Manifest

```diff
@@ -0,0 +1,2 @@
+image flannel-flannel docker.io/flannel/flannel:v0.20.2
```

Comment:
This is still using the wrong version.

Reply:
The script re-run and tghe values were updated again we will need to create a PR based on this one and probably see how to fix the script as well.